campaignion_tracking/campaignion_tracking.info

```diff
@@ -2,3 +2,6 @@ name = Campaignion Tracking
 description = Common tracking functionality.
 core = 7.x
 package = Campaignion
+
+dependencies[] = campaignion:campaignion_action
+dependencies[] = little_helpers:little_helpers (>=2.0-alpha7)
```

Comment:
A dependency on webform_paymethod_select:webform_paymethod_select is missing too.